.github/actions/javascript/authorChecklist/authorChecklist.ts

```diff
@@ -132,7 +124,8 @@ async function generateDynamicChecksAndCheckForCompletion() {
             }
         }
     }
-    const allChecks = _.flatten(_.map(_.values(CHECKLIST_CATEGORIES), 'items'));
+    // eslint-disable-next-line you-dont-need-lodash-underscore/flatten, you-dont-need-lodash-underscore/map
```

Comment:
let's try to avoid suppressing the linter, it's there for a reason:

```js
const allChecks = Object.values(CHECKLIST_CATEGORIES)
    .map(category => category.items)
    .flat();
```

This rule in particular is especially valuable for GitHub Actions if it avoids us bundling `lodash` or `underscore` in the final executable output of `ncc`